src/cljc/rems/common/application_util.cljc

```diff
@@ -27,3 +29,15 @@
 
 (defn is-handler? [application user]
   (contains? (workflow-handlers application) user))
+
+(defn- str-to-int [s]
+  #?(:clj (Integer/parseInt s)
+     :cljs (js/parseInt s 10)))
+
+(defn parse-sequence-number-within-year-to-number [external-id]
+  (str-to-int (get (str/split external-id #"/") 1)))
```

Comment:
This function assumes that the format will always be `year/number` and the external-id could be something like `THL/2492/4.00.00/2020` so this should be written a bit more generic, and the name would then change. I would name it  something like `parse-sortable-external-id`